spec/factories/quotes.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :quote do
+    author   Faker::Name.first_name
+    body     Faker::Lorem.sentences.join(" ")
```

Comment:
We tend to avoid column aligning text like this as it ends up creating more diffs in commits than were necessary.